manage/sawtooth_manage/subproc.py

```diff
@@ -103,13 +103,14 @@ def start(self, node_args):
             if cmd == 'validator':
                 component = '--component-endpoint', url
                 network = '--network-endpoint', gossip_port
+                public_uri = '--public-uri', gossip_port
                 peer_list = ['tcp://0.0.0.0:' + str(base_gossip_port + i)
```

Comment:
Noted. Subprocess was originally coded with 0.0.0.0 as the network address, but I have changed it to "localhost," after consultation with Boyd and others.